src/prpy/planning/ompl.py

```diff
@@ -359,21 +362,23 @@ class OMPLSimplifier(BasePlanner):
     def __init__(self):
         super(OMPLSimplifier, self).__init__()
 
-        planner_name = 'OMPL_Simplifier'
-        self.planner = openravepy.RaveCreatePlanner(self.env, planner_name)
-
-        if self.planner is None:
-            raise UnsupportedPlanningError(
-                'Unable to create OMPL_Simplifier planner.')
-
     def __str__(self):
         return 'OMPL Simplifier'
 
-    @ClonedPlanningMethod
+    @LockedPlanningMethod
     def ShortcutPath(self, robot, path, timeout=1., **kwargs):
         # The planner operates in-place, so we need to copy the input path. We
         # also need to copy the trajectory into the planning environment.
-        output_path = CopyTrajectory(path, env=self.env)
+
+        planner_name = 'OMPL_Simplifier'
+        env = robot.GetEnv()
+        planner = openravepy.RaveCreatePlanner(env, planner_name)
+
+        if planner is None:
+            raise UnsupportedPlanningError(
+                'Unable to create OMPL_Simplifier planner.')
```

Comment:
Nit: Modify this message use `planner_name` so this message can't get out of sync with the code.